### PR TITLE
Dseisun/change layout

### PR DESCRIPTION
**Pull request by dseisun**
Change the package layout to be more in line with python standards so that Craig from work stops making fun of me

### CODE REVIEW

**Comment by dseisun** [comment]
Relates to issue #25

**Comment by dseisun** [comment]
moving forward since it's really annoying to keep my branches in sync with this change

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from distutils.core import setup
+
+setup(
+    name='nerdouts',
+    version='0.1dev',
```
Comment (jnguyen75) [inline setup.py]: FWIW [PEP 440](https://www.python.org/dev/peps/pep-0440/#developmental-releases) specifies `X.Y.devN` for version

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    packages=['nerdouts'],
+    license='Creative Commons Attribution-Noncommercial-Share Alike license',
```
Comment (jnguyen75) [inline setup.py]: TIL about this license..

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    scripts=['nerdouts/workout'],
+    data_files=[('.', ['README.md'])],
+    long_description=open('README.md').read(),
```
Comment (jnguyen75) [inline setup.py]: Neat trick.  I usually put the documentation at the top of the file use `__doc__`

File: setup.py (hunk 1 above)
Comment (dseisun) [reply setup.py]: lol, good catch

File: setup.py (hunk 2 above)
Comment (dseisun) [reply setup.py]: Same. This was definitely carefully chosen and not copy/pasted...

File: setup.py (hunk 3 above)
Comment (dseisun) [reply setup.py]: oh interesting. Actually think I might prefer that

**Pull request merged**